PlayerKit/Classes/Player.swift

```diff
@@ -124,16 +124,26 @@ public enum PlayerError: Int
 {
     var displayName: String { get }
     var locale: Locale? { get }
-    var describesMusicAndSound: Bool { get }
+    var isSDHTrack: Bool { get }
```

Comment:
Great change. It might be valuable to include documentation here that says something like, 

```Swift
/// Indicates the text track represents subtitles for the deaf and hard of hearing (SDH).
```

Just to make it clear what SDH is.

Reply:
Yeah good call. I'll pop that in before merging.